docs/metrics.md

```diff
@@ -0,0 +1,14 @@
+# Build Controller Metrics
+
+The Build component exposes several metrics to help you monitor the health and behavior of your build resources.
+
+Following build metrics are exposed at service `build-operator-metrics` on port `8383`.
+
+| Name | Type | Description | Status |
+| ---- | ---- | ----------- | ------ |
+| `up_status` | Gauge | The up status of the build-controller. | experimental |
+| `build_count` | Gauge | Build creation count. | experimental |
+| `buildrun_count` | Gauge | BuildRun execution count. | experimental |
+| `buildrun_establish_duration` | Gauge | BuildRun establish duration in seconds. | experimental |
```

Comment:
Can you explain in the help text what "established" means?

Reply:
`buildRunEstablishDuring = buildRun.Status.StartTime.Time - buildRun.CreationTimestamp.Time`

It is the preparation time in buildrun controller logic

Reply:
That makes a lot of sense, can you reflect that in the help text? :)